src/appfl/config/config.py

```diff
@@ -22,8 +22,16 @@ class Config:
     test_data_batch_size: int = 64
     test_data_shuffle: bool = False
 
-    # Results directory
-    result_dir: str = "./results"
+    # Initial Model Parameters
+    is_init_point: bool = True
```

Comment:
The default value would better be `False`, and the values below be empty string.